### PR TITLE
Added travis merge

### PR DESCRIPTION
**Pull request by SchSeba**
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/40)

### CODE REVIEW

File: stdci.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+archs: x86_64
```
Comment (phoracek) [inline stdci.yaml]: Why do you change stdci.yaml and check-merged? in Travis PR?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+deploy:
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push
+  skip_cleanup: true
+  on:
+    branch: master
+- provider: script
+  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push IMAGE_TAG=$TRAVIS_TAG
+  skip_cleanup: true
+  file:
```
Comment (phoracek) [inline .travis.yml]: Why is this empty?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+dist: trusty
+
+services:
+- docker
+
+env:
+- BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
+
+install:
+ - git reset --hard
```
Comment (phoracek) [inline .travis.yml]: Why do we need this?

Comment (SchSeba) [reply .travis.yml]: Use the kubevirt as a base file
https://github.com/kubevirt/kubevirt/blob/master/.travis.yml#L11

File: .travis.yml (hunk 2 above)
Comment (SchSeba) [reply .travis.yml]: Use the kubevirt as a base file

https://github.com/kubevirt/kubevirt/blob/master/.travis.yml#L37

File: stdci.yaml (hunk 1 above)
Comment (SchSeba) [reply stdci.yaml]: my bad make a checkout from another PR and not from master.

File: .travis.yml (hunk 3 above)
Comment (phoracek) [reply .travis.yml]: We should understand *why* first. Same for the empty "file" section.

**Comment by SchSeba** [comment]
@phoracek Can you take a look please?

**Comment by phoracek** [comment]
LGTM

**Pull request merged**